### DISCUSSION TITLE
Contributing with Laravel docs

### DISCUSSION THREAD

**Discussion by Arkanius** [laravel/framework#32870] [General] [answered]
Hello everyone, 

I've a simple doubt about how to contribute with docs. Currently, laravel/docs has the branch 7.x and master (without mention the others). When a feature is merged at master (8.x) branch of laravel/framework should I use the master branch of the docs as well?

Tks

**Comment by driesvints** [comment, marked as answer]
Yes. All Laravel 8 features should target the master branch.
